lti_consumer/lti_consumer.py

```diff
@@ -155,7 +155,7 @@ def parse_handler_suffix(suffix):
 LaunchTargetOption = namedtuple('LaunchTargetOption', ['display_name', 'value'])
 
 
-class LaunchTarget(object):
+class LaunchTarget:  # pylint: disable=bad-option-value, old-style-class
```

Comment:
Same as above, we will need this to run in both python2 and python3 for a little bit.